Source/SwiftLintFramework/Rules/Lint/AnyObjectProtocolRule.swift

```diff
@@ -12,7 +12,6 @@ public struct AnyObjectProtocolRule: SubstitutionCorrectableASTRule, OptInRule,
         name: "AnyObject Protocol",
         description: "Prefer using `AnyObject` over `class` for class-only protocols.",
         kind: .lint,
-        minSwiftVersion: .fourDotOne,
```

Comment:
this is a warning in the compiler now - depending when it was added (need to check) we should remove this rule

Reply:
looks like it was added in 5.4.1 https://github.com/apple/swift/commit/e3373a9efd9c9f01757f1d9878ec9981722e669c, so let's keep it for now